e2e/tests/utils/openAppForDebugBuild.js

```diff
@@ -0,0 +1,31 @@
+const appConfig = require('../../../app.json');
+
+module.exports.openAppForDebugBuild = async function openAppForDebugBuild() {
+  const [platform, target] = process.env.DETOX_CONFIGURATION.split('.');
```

Comment:
What is the advantage of this over `detox.getConfigurationName()`, `detox.getPlatform()`, etc.?

Reply:
The disadvantage of those is that I didn't know they existed. AFAIR you also didn't used them in your code 😅

Reply:
Those don't exist...

<img width="679" alt="Screenshot 2022-10-11 at 09 20 11" src="https://user-images.githubusercontent.com/5256730/195021816-6928015f-112c-4c77-a945-847545910e46.png">
<img width="683" alt="Screenshot 2022-10-11 at 09 19 12" src="https://user-images.githubusercontent.com/5256730/195021826-d78d86b5-5266-431e-83fd-d761d65bb682.png">

I'm even more confused now.